### ISSUE TITLE
weirdness with mpd server and autoaudiosync + shout2send

### ISSUE THREAD

**Issue by lilmike** [mopidy/mopidy#1535]
Hi,
I have an output setting in mopidy.conf which looks like:

output = tee name=t t. ! queue ! audioconvert ! audioresample ! autoaudiosink t. ! queue ! audioresample ! audioconvert ! vorbisenc ! oggmux ! shout2send [...]

Now, when the track changes on its own, the mpd server, using mpc, reports the original song it started on. This continues--not changing the name, etc of the song--until I manually stop and restart the playback, or when I change a track manually. It might not be changed somewhere else too, I'm not sure, but mpd/mpc is where I notice it.
Thanks,
-Michael.

**Comment by lilmike** [comment]
Ok, also, it seems that the mpd server isn't actually affecting mopidy in some ways either, as I set repeat and random on, and then turned off random, and when I don't change the track myself, sometimes (seemingly not always?) it plays the same track over and over. Changing track manually will make it play different songs for a few, but then it gets stuck again. I've verified that mpd random is off, but obviously on mopidy's... other end? it's not.
-Michael.
